src/kOS.Safe/Compilation/CompiledObject.cs

```diff
@@ -580,9 +581,8 @@ public static List<CodePart> UnPack(GlobalPath path, string prefix, byte[] conte
         /// </summary>
         /// <param name="program">recently built program parts to re-assign.</param>
         /// <param name="path">path of file (with preceeding volume:) that the compiled code came from, for rutime error reporting purposes.</param>
-        /// <param name="prefix">a string to prepend to the labels in the program.</param>
         /// <param name="lineMap">describes the mapping of line numbers to code locations.</param>
-        private static void PostReadProcessing(IEnumerable<CodePart> program, GlobalPath path, string prefix, DebugLineMap lineMap)
+        private static void PostReadProcessing(IEnumerable<CodePart> program, GlobalPath path, DebugLineMap lineMap)
```

Comment:
I was looking for where I had to make changes for how the ``prefix`` arg was being used, and discovered it's actually never used in the body of this method at all.  By the time this is called, the ``prefix`` is just irrelevant now - the relabeling has already been done earlier by ``ReadOpcodeList()`` by this point. 

I think this *used to be* where the prefix was added, but that logic had to be moved quite a while ago, and when it was moved, this never got changed.